Compiled from "AuthException.java"
public class net.oneandone.stool.docker.AuthException extends java.io.IOException {
  public final java.lang.String realm;

  public final java.lang.String service;

  public final java.lang.String scope;

  public net.oneandone.stool.docker.AuthException(java.lang.String, java.lang.String, java.lang.String);
    Code:
       0: aload_0
       1: aload_1
       2: aload_2
       3: aload_3
       4: invokedynamic #1,  0              // InvokeDynamic #0:makeConcatWithConstants:(Ljava/lang/String;Ljava/lang/String;Ljava/lang/String;)Ljava/lang/String;
       9: invokespecial #5                  // Method java/io/IOException."<init>":(Ljava/lang/String;)V
      12: aload_0
      13: aload_1
      14: putfield      #11                 // Field realm:Ljava/lang/String;
      17: aload_0
      18: aload_2
      19: putfield      #17                 // Field service:Ljava/lang/String;
      22: aload_0
      23: aload_3
      24: putfield      #20                 // Field scope:Ljava/lang/String;
      27: return
}
